[HDRP] Adding information about SSGI and Emissive incompatibility in documentation and override (#5468)
Unity-Technologies/Graphics: com.unity.render-pipelines.high-definition/Documentation~/Override-Screen-Space-GI.md

## Limitations

> In Deferred rendering mode, Screen Space Global Illumination and Ray-Traced Global Illumination share a buffer with emissive thus overwriting emissive data. 

Reviewer: ```suggestion
In Deferred rendering mode, Screen Space Global Illumination and Ray-Traced Global Illumination share a buffer with emissive, which overwrites emissive data. 
```

> There is multiple ways to recover the emissive contribution of the scene materials: 

Reviewer: ```suggestion
There are multiple ways to recover the emissive contribution of the scene materials: 
```

> * Disabling [Receive SSR/SSGI](snippets/shader-properties/surface-options/receive-ssr.md) flag on the emissive materials.

Reviewer: ```suggestion
* Disable [Receive SSR/SSGI](snippets/shader-properties/surface-options/receive-ssr.md) flag on the emissive materials.
```

> * Using [Force Forward Emissive](snippets/shader-properties/advanced-options/force-forward-emissive.md) on the emissive materials.

Reviewer: ```suggestion
* Use [Force Forward Emissive](snippets/shader-properties/advanced-options/force-forward-emissive.md) on the emissive materials.
```

> * Using Forward rendering.

Reviewer: ```suggestion
* Use Forward rendering.
```
